client/js/components/map/admin/MapAdmin.vue

```diff
@@ -294,7 +300,7 @@ export default {
   },
 
   async mounted () {
-    const settings = await this.fetchProcedureMapSettings(this.procedureId)
+    const settings = await this.fetchProcedureMapSettings({ procedureId: this.procedureId, isMaster: this.isMaster })
```

Comment:
the method is also used here:
client/js/components/procedure/StatementSegmentsList/StatementSegmentsList.vue
If we change the structure to an Object, we have to change that there too.

Reply:
Thanks for catching, done here [6e61bac](https://github.com/demos-europe/demosplan-core/pull/3738/commits/6e61bac7c3e0defe817f0d323c71bc4843506680)